translator/tocwconfig/sampleConfig/complete_linux_config.yaml

```diff
@@ -1,362 +1,400 @@
+    encoding: utf-8
+    id: udp_input
+    listen_address: 127.0.0.1:25888
+    multiline:
+      line_end_pattern: .^
+      line_start_pattern: ""
+      omit_pattern: false
+    operators: [ ]
+    output: [ ]
+    resource: { }
+    retry_on_failure:
+      enabled: false
+      initial_interval: 0s
+      max_elapsed_time: 0s
+      max_interval: 0s
+    storage: null
+    type: udp_input
 service:
-    extensions:
-        - agenthealth/metrics
-        - agenthealth/logs
-        - agenthealth/traces
-    pipelines:
-        logs/emf_logs:
-            exporters:
-                - awscloudwatchlogs/emf_logs
-            processors:
-                - batch/emf_logs
-            receivers:
-                - udplog/emf_logs
-        metrics/host:
-            exporters:
-                - awscloudwatch
-            processors:
-                - ec2tagger
-                - transform
-            receivers:
-                - telegraf_mem
-                - telegraf_socket_listener
-                - telegraf_processes
-                - telegraf_procstat/1917393364
-                - telegraf_disk
-                - telegraf_swap
-                - telegraf_netstat
-                - telegraf_cpu
-                - telegraf_statsd
-                - jmx/0
-                - jmx/1
-        metrics/hostDeltaMetrics:
-            exporters:
-                - awscloudwatch
-            processors:
-                - cumulativetodelta/hostDeltaMetrics
-                - ec2tagger
-                - transform
-            receivers:
-                - telegraf_net
-                - telegraf_diskio
-        traces/xray:
-            exporters:
-                - awsxray
-            processors:
-                - batch/xray
-            receivers:
-                - awsxray
-                - otlp
-    telemetry:
-        logs:
-            development: false
-            disable_caller: false
-            disable_stacktrace: false
-            encoding: console
-            error_output_paths: []
-            initial_fields: {}
-            level: error
-            output_paths:
-                - /tmp/fake/log/hotdog.log
-            sampling:
-                enabled: true
-                initial: 2
-                thereafter: 500
-                tick: 10s
-        metrics:
-            address: ""
-            level: None
-            readers: []
-        resource: {}
-        traces:
-            processors: []
-            propagators: []
+  extensions:
+    - agenthealth/metrics
+    - agenthealth/logs
+    - agenthealth/traces
+  pipelines:
+    logs/emf_logs:
+      exporters:
+        - awscloudwatchlogs/emf_logs
+      processors:
+        - batch/emf_logs
+      receivers:
+        - udplog/emf_logs
+    metrics/jmx/0:
+      exporters:
+        - awscloudwatch
+      processors:
+        - filter/jmx/0
+      receivers:
+        - jmx/0
+    metrics/jmx/1:
+      exporters:
+        - awscloudwatch
+      processors: [ ]
```

Comment:
Should this have ec2 tagger like metrics?

Reply:
This pipeline needs to function the same as our regular metrics pipeline

Reply:
Maybe instead of having an exporter to CW we can use https://opentelemetry.io/docs/collector/building/connector/ to connect to regular metrics pipeline

Reply:
added the` ec2tagger` to the jmx pipelines